### ISSUE TITLE
RFE: Please include output from 'docker build' 

### ISSUE THREAD

**Issue by maxamillion** [projectatomic/atomic-reactor#281]
As per Tomas' request in the [mailing list thread](https://lists.projectatomic.io/projectatomic-archives/atomic-devel/2015-August/msg00034.html), I am filing this RFE.

Currently the log information when using the atomic-reactor API is limited in scope to what happens from inside the build container:
```python
    response = \
       build_image_using_hosts_docker(
            "buildimage",
            source,
            "test-build"
        )
    response.build_logs
```

I would like to request that the external information also be available as log information, similar to:
```
# docker build .
Sending build context to Docker daemon   105 kB
Sending build context to Docker daemon
Step 0 : FROM fedora:latest
 ---> ded7cd95e059
Step 1 : MAINTAINER http://fedoraproject.org/wiki/Cloud
 ---> Using cache
 ---> 9f97f024d47a
Step 2 : RUN dnf -y update && dnf clean all
 ---> Using cache
 ---> e5b0ea2b7754
Step 3 : RUN dnf -y install httpd && dnf clean all
 ---> Using cache
 ---> 0c9f137f508c
Step 4 : RUN echo "Apache" >> /var/www/html/index.html
 ---> Using cache
 ---> 85461d238aef
Step 5 : EXPOSE 80
 ---> Using cache
 ---> 7c08c855c39f
Step 6 : ADD run-apache.sh /run-apache.sh
 ---> 83f8d7985d04
Removing intermediate container c3107d502888
Step 7 : RUN chmod -v +x /run-apache.sh
 ---> Running in e0cce7c17116
mode of '/run-apache.sh' changed from 0644 (rw-r--r--) to 0755 (rwxr-xr-x)
 ---> 93c019e967d6
Removing intermediate container e0cce7c17116
Step 8 : CMD /run-apache.sh
 ---> Running in 6bd5a6cfc7f3
 ---> e6d701d7f85e
Removing intermediate container 6bd5a6cfc7f3
Successfully built e6d701d7f85e
```

Thank you,
-AdamM

**Comment by twaugh** [comment]
This is currently stored as an OSv3 Build annotation. With the new Content Generator workflow, however, we should instead store this log file as a separate output attached to the Koji Build.

**Comment by vrutkovs** [comment]
This is resolved in #319 - build logs are stored in http://<koji>/<kojiroot>/packages/<package>/<version>/<release>/data/logs/build.log
